memory/ndctl.py.data/ndctl.yaml

```diff
@@ -2,6 +2,8 @@ config:
     namespace_cnt: 4
     region: 'region0'
     preserve_change: True
+    modes:
```

Comment:
@harish-24 does it makes sense to have the accepted modes multiplexed here, instead of using just one value from user ?
If yes, does the same applicable for size as well ?

Reply:
Yes, that is the idea. But did not want to expand with yaml here as I considered this yaml file just as a placeholder.

Reply:
@harish-24 can you explain more please ?

Reply:
@narasimhan-v modes/size can be multiplexed. I added it here just to let the user know these options can be inputs. It really depends on the user whether it needs to be multiplexed or not. Okay, so I will add these are !mux inputs rather than one value so that user can have a better option to input.